optimum/intel/openvino/quantization.py

```diff
@@ -212,6 +232,14 @@ def quantize(
         else:
             raise TypeError(f"Unsupported model type: {type(self.model)}")
 
+    def _get_compression_options(self, config: OVConfig):
```

Comment:
FYI because `nncf.CompressWeightsMode` is a string `Enum`, you can do convert the string to the Enum like so: 

```python
mode = nncf.CompressWeightsMode["int4_sym"]
```
So, instead of the table approach you could create the options dictionary by simply parsing the different components of the `config.compression` string.

Reply:
the names in the compression_option and in the nncf.CompressWeightsMode are different anyway. Moreover, we introduce some experimental options in the nncf.CompressWeightsMode that we don't want to expose until they are not fully functional with OpenVINO. So, it makes sense to keep the mapping.